### PR TITLE
Canada geometry names

### PR DESCRIPTION
**Pull request by michellemho**
Adding geometry names for Canadian provinces + territories, census tracts, census divisions, census subdivisions, and census metropolitan areas and census agglomerations.

Assuming that all name codes are "RESOLUTION+NAME" (eg. "CTNAME")

### CODE REVIEW

**Comment by talos** [comment]
Looks like TIGER changes slipped in here.  Will have to hold off on review until #153 changes are merged.

**Comment by ethervoid** [comment]
Is this ok? after fixing the conflic is ok to merge?

**Comment by michellemho** [comment]
I believe so @ethervoid.

**Comment by ethervoid** [comment]
To clarify for a newbie hehe, what is the purpose of including this names?

**Comment by michellemho** [comment]
A while back, John and I added in a "Proper Names" subsection tag so we could log the proper names of geometries. Most of the lower level geographies (for example Census Tracts in the US) do not have proper names. But higher level geographies such as Provinces or States will have proper names. We wanted to capture those.

**Comment by michellemho** [comment]
The process was to look at the shapefile or original raw data of the downloaded shapes. Usually, that data comes with a proper name, as well as a "geo_id" column which we are already adding.

All I needed to do was see what the proper name column was labeled in the original raw data. In the case of Canada, it appears the proper names in the geography tables were labeled 'CTNAME', 'PRNAME', 'CDNAME', 'CSDNAME', and 'CMANAME'. I would refer to these labels in the `populate` method of the TableTask.

I would also add proper names to the Geography Columns task, giving it a target type of  `targets={geom: GEOM_NAME}`.

Sometimes, the proper name is exactly the same as the geo_id (usually the case for smaller geographies). I still added them; even if it means duplicate information.

File: tasks/ca/statcan/geo.py [hunk 1]
```diff
@@ -160,9 +177,11 @@ def columns(self):
     def populate(self):
         session = current_session()
         session.execute('INSERT INTO {output} '
-                        'SELECT {code}uid as geom_id, '
+                        'SELECT {name} as geom_name, '
+                        '       {code}uid as geom_id, '
                         '       wkb_geometry as geom '
                         'FROM {input} '.format(
+                            name=GEOGRAPHY_PROPERNAMES[self.resolution()],
```
Comment (antoniocarlon) [inline tasks/ca/statcan/geo.py]: This line should be `name=GEOGRAPHY_PROPERNAMES[self.resolution],` (removing the parentheses)

**Pull request merged**